src/styles/blocks/header.scss

```
@@ -0,0 +1,62 @@
+
+.header {
+  // margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo-link-block {
+    padding: 10px 0;
+  }
+
+  &__logo-img {
+    display: block;
+  }
+
+  &__link-list {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    display: flex;
+  }
+
+  &__list-item {
+    display: inline-block;
+    &:not(:last-of-type) {
+      margin-right: 20px;
+    }
+  }
+
+  &__list-link {
+    font-weight: 500;
+    font-size: 12px;
+    color: $main-accent;
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 23px 0;
+
+    &:nth-child(4) {
+      color: #00acdc;
+    }
+
+    &:hover {
+      color: #00acdc;
+    }
+
+    &--active {
+      color: #00acdc;
+      position: relative;
+      &::after {
+        content: "";
+        position: absolute;
+        height: 4px;
+        width: 100%;
+        background-color: #00acdc;
```

this color uses more than one time
create variable for it 
check other similar cases

---

saw that already exists variable for such color. just use it
```suggestion
        background-color: $blue-accent;
```